### PR TITLE
Deprecate EIP-777

### PR DESCRIPTION
**Pull request by Pandapip1**
## Description

Adds a deprecation notice next to the mention of EIP-777 on the standards page.

## Related Issue

N.A.

### CODE REVIEW

**Comment by Pandapip1** [comment]
EIP-777 is tricky to implement correctly (it's particularly vulnerable to reentrancy attacks) and is not widely used. OpenZeppelin is planning on [hiding their EIP-777 implementation from the documentation](https://github.com/OpenZeppelin/openzeppelin-contracts/issues/2620) in an event to deprecate it.

**Comment by minimalsm** [comment]
Thanks @Pandapip1. Curious if you think we should still have a page on EIP-777?

**Comment by Pandapip1** [comment]
I think we should, even as a historical archive.

**Comment by Pandapip1** [comment]
Bump.

File: src/content/developers/docs/standards/tokens/erc-777/index.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ lang: en
 sidebar: true
 ---
 
+## Deprecation Notice {#deprecation-notice}
+
+**ERC-777 is deprecated. It is recommended to use [ERC-20](/developers/docs/standards/tokens/erc-20/) instead.** This page remains as a historical archive.
```
Comment (axic) [inline src/content/developers/docs/standards/tokens/erc-777/index.md:10]: I think this is a strong statement and makes ethereum.org very opinionated. Who decides "deprecation"?

A better way would be pointing to some "expert discussion" stating "ERC-777 is not recommended by some/many due to potential security issues".

Comment (Pandapip1) [reply src/content/developers/docs/standards/tokens/erc-777/index.md:10]: I'll change the wording.

**Review by minimalsm — DISMISSED** [review]

**Pull request merged**